modules/mod_ginger_tagger/mod_ginger_tagger.erl

```diff
@@ -19,18 +19,20 @@
 ]).
 
 manage_schema(install, Context) ->
```

Comment:
Version is `install`?

Context -> _Context (unused)

Reply:
Version type signature is:

```
-type manage_schema() :: install
                       | {upgrade, integer()}.
```